lib/active_model/serializer/attribute.rb

```diff
@@ -0,0 +1,38 @@
+module ActiveModel
+  class Serializer
+    Attribute = Struct.new(:name, :block, :key, :condition_type, :condition) do
+      def initialize(name, block, options)
+        super(name, block)
```

Comment:
since the `:key, :condition_type, :condition` attributes to an initializer lumped as `options` (which I initially missed), why not just take in options and create our own accessors for key, condition_type, condition?